### PR TITLE
Implement reselectors for TimeEntries(filter& sort) and TeamMembers(sort). 

### PR DESCRIPTION
**Pull request by RikFrieling**

### CODE REVIEW

File: ducks/team-members.js [hunk 1]
```diff
@@ -47,3 +85,8 @@ export const requestTeamMembers = () => ({ type: REQUEST_TEAM_MEMBERS });
 export const requestTeamMembersSucces = (teamMembers) => (
   { type: REQUEST_TEAM_MEMBERS_SUCCESS, teamMembers }
 );
+
+export const sortTeamMembersSuccess = (sortBy) => ({ type: SORT_TEAM_MEMBERS_SUCCESS, sortBy });
+export const changeSortDirectionSuccess = (sortDirection) => (
```
Comment (jellebouwman) [inline ducks/team-members.js]: This function, and the one above, imply there is an action that was called before. I think using `success` in the names of your action creators should only be used when you when you have a concrete action that was called before this one. For example with `requestTeamMembers`and `requestTeamMembersSuccess`.

File: ducks/team-members.js [hunk 2]
```diff
@@ -47,3 +85,8 @@ export const requestTeamMembers = () => ({ type: REQUEST_TEAM_MEMBERS });
 export const requestTeamMembersSucces = (teamMembers) => (
```
Comment (jellebouwman) [inline ducks/team-members.js]: Success with two `s`'s

File: ducks/team-members.js [hunk 3]
```diff
@@ -29,6 +65,8 @@ export const teamMembersReducer = (state = initialState, action) => {
       return { ...state, isLoading: true };
     case REQUEST_TEAM_MEMBERS_SUCCESS:
       return { ...state, items: action.teamMembers, isLoading: false };
+    case SORT_TEAM_MEMBERS_SUCCESS:
+      return { ...state, sortBy: action.sortBy, isLoading: false };
```
Comment (jellebouwman) [inline ducks/team-members.js]: You set `isLoading` to `false`, but never set it on `true`. Also happens on line 55 with `CHANGE_SORT_DIRECTION_SUCCES`

File: components/time-entry-overview/TimeEntryOverview.jsx [hunk 4]
```diff
@@ -30,19 +32,50 @@ class TimeEntryOverview extends React.Component {
...
-          timeEntries={timeEntries}
-          onDelete={this.onDelete}
-        />
+      <div className="time-entry-overview">
+        <div className="time-entry-overview__header">
+          <div className="time-entry-overview__header-title">
+            <h1 className="time-entry-overview__header-text"> Timesheets </h1>
+            <h2 className="time-entry-overview__header-summation"> 12 entries </h2>
+          </div>
+          <div className="time-entry-overview__header-search-and-select">
```
Comment (jellebouwman) [inline components/time-entry-overview/TimeEntryOverview.jsx]: Do you need to wrap these two input elements with this div? You could set the `margin-left` of the select element to `auto` to allign both the select and input element to the right.

File: ducks/team-members.js (hunk 2 above)
Comment (RikFrieling) [reply ducks/team-members.js]: Ssi

File: components/time-entry-overview/TimeEntryOverview.jsx (hunk 4 above)
Comment (RikFrieling) [reply components/time-entry-overview/TimeEntryOverview.jsx]: changed name in left-side and right side

**Pull request merged**